pages/home/EmailSignupForm.jsx

```diff
@@ -20,10 +20,10 @@ var PledgeSignupForm = React.createClass({
       return;
     }
 
-    if (process.env.NODE_ENV !== 'production' && !process.env.PLEDGE_MAILINGLIST_URL) {
+    if (process.env.NODE_ENV !== 'production' && !process.env.NEWSLETTER_MAILINGLIST_URL) {
       e.preventDefault();
-      alert("PLEDGE_MAILINGLIST_URL is not defined. Simulating a successful pledge signup now.");
-      window.location = "?pledge=thanks";
+      alert("NEWSLETTER_MAILINGLIST_URL is not defined. Simulating a successful newsletter signup now.");
+      window.location = "?signup=thanks";
```

Comment:
I'll need to adjust form config on BSD for this url change.

@adamlofting can we still make changes to a BSD sign form? Namely changing the thank you redirect URL for the "Mentor Signup Form".